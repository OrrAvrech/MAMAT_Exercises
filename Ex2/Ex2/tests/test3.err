Command not found.
Add execution failed.
Insert failed: not enough parameters.
Add execution failed.
Insert execution failed.
Insert execution failed.
Insert execution failed.
Add failed: not enough parameters.
Add execution failed.
Add execution failed.
Add execution failed.
Add execution failed.
Depart execution failed.
