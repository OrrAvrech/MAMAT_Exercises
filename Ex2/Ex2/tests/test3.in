Mister Boin
Add 22 D JFK R
Insert 1 
Insert 1 D
            Insert 2 D
Add 23 I JFK R
Insert 2 D
Insert 2 I
Insert 3 R
Insert 3 I
Insert 4 I
Add 22 D JFK 
Add 22 D JFK R
Add 22 D JFK R
Add 23 D JFKR R
Add 23 P JFK R
Add -2 D HYT R
Add 24 D JFK R
Add 26 D ABC R
Add 27 D DBZ R
Add 28 D JFK E
Add 29 D JFK E
Add 31 D ABC E
Add 32 I ABC E
Add 33 I ABC R
Print
Remove 3
Change JFK POK
Delay POK
Print
Depart
Depart
Print
Depart
Depart
Print
Depart
Depart
Print
Depart
Depart
Depart
Delay JFK
Change JFK POK