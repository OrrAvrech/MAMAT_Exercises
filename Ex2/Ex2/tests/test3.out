Airport status:
Runway 1 domestic
4 flights are waiting:
Flight 28 D JFK E
Flight 31 D ABC E
Flight 22 D JFK R
Flight 26 D ABC R
Runway 2 domestic
3 flights are waiting:
Flight 29 D JFK E
Flight 24 D JFK R
Flight 27 D DBZ R
Runway 3 international
1 flights are waiting:
Flight 32 I ABC E
Runway 4 international
1 flights are waiting:
Flight 33 I ABC R

Airport status:
Runway 1 domestic
4 flights are waiting:
Flight 31 D ABC E
Flight 28 D POK E
Flight 26 D ABC R
Flight 22 D POK R
Runway 2 domestic
3 flights are waiting:
Flight 29 D POK E
Flight 27 D DBZ R
Flight 24 D POK R
Runway 4 international
1 flights are waiting:
Flight 33 I ABC R

Airport status:
Runway 1 domestic
2 flights are waiting:
Flight 26 D ABC R
Flight 22 D POK R
Runway 2 domestic
3 flights are waiting:
Flight 29 D POK E
Flight 27 D DBZ R
Flight 24 D POK R
Runway 4 international
1 flights are waiting:
Flight 33 I ABC R

Airport status:
Runway 1 domestic
1 flights are waiting:
Flight 22 D POK R
Runway 2 domestic
2 flights are waiting:
Flight 27 D DBZ R
Flight 24 D POK R
Runway 4 international
1 flights are waiting:
Flight 33 I ABC R

Airport status:
Runway 1 domestic
0 flights are waiting:
Runway 2 domestic
1 flights are waiting:
Flight 24 D POK R
Runway 4 international
1 flights are waiting:
Flight 33 I ABC R

